### PR TITLE
Add blessings

### PR DESCRIPTION
**Pull request by synapticarbors**

### CODE REVIEW

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes This PR is ready for review.

File: recipes/blessings/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (jakirkham) [inline recipes/blessings/meta.yaml]: Why skip Windows? I know one needs something like `colorama` as well, but shouldn't this still work on Windows?

Comment (synapticarbors) [inline recipes/blessings/meta.yaml]: I was basing that decision on the following issue that seems to indicate that running on windows isn't going to work:

https://github.com/erikrose/blessings/issues/21

Also:

https://github.com/erikrose/blessings/blob/master/setup.py#L37

**Comment by jakirkham** [comment]
Thanks @synapticarbors.

**Pull request merged**